TAF_EasyRest/TestFramework/Test/Sample test.cs

```diff
@@ -0,0 +1,58 @@
+namespace TestFramework.Test
+{
+    [TestFixture]
+    public class SampleTests
+    {
+        IWebDriver Chromedriver;
+        private static string password;
+        private static string email;
+        private static string baseUrl;
+
+        [OneTimeSetUp]
+        public void BeforeAllTests()
+        {
+            Chromedriver = new ChromeDriver();
+            baseUrl = "http://localhost:3000"; //move to json
+        }
+
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void PositiveTestLoginToEasyrest()
+        {
+            //Arrange
+            SignInPage page = new SignInPage(Chromedriver);
+            string expected = $"{baseUrl}/restaurants";
+            password = "1";
+            email = "eringonzales@test.com";
+
+            //Act
+            page.GoToUrl();
+            page.ClickEmailField()
+                .SendKeysToEmailField(email)
+                .ClickPasswordField()
+                .SendKeysToPasswordField(password)
+                .ClickSignInButton();
```

Comment:
I think you can Sign In steps move to Setup()

Reply:
agree, this test was added only to check waiters. it will be deleted soon.